pkg/config/config.go

```diff
@@ -0,0 +1,48 @@
+package config
```

Comment:
created this file and the package to hold the config for the operator.

I wanted operator to receive things like image stream details etc. via env vars. In the end they were too many and decided to create this single file which holds all config.